.github/workflows/dotnet.yml

```diff
@@ -33,14 +33,14 @@ jobs:
               run: dotnet pack -c Release -o out
 
             - name: Push generated package to NuGet registry
-              if: github.ref == 'refs/heads/main'
-              run: dotnet nuget push out/*.nupkg -k $NUGET_AUTH_TOKEN -s https://api.nuget.org/v3/index.json
+              # if: github.ref == 'refs/heads/main'
```

Comment:
Just curious, is this just a temporary workaround to force republishing?

Reply:
yup - have a TMP commit d1f9cf16a7e887cff6321b1d40ea1cacbffcee09 that i'll revert before merging